3.Linear_Modeling/1a.fit_linear_model_1beta.ipynb

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    from typing import Union
</pre><p> </p><p>Try moving this at the top of where all the standard libraries are being imported. <code>typing</code> is part of the python's standard library   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>